draft/draft-ietf-netconf-https-notif.xml

```diff
@@ -204,12 +203,14 @@
             </artwork>
           </figure>
         </t>
+	<t>The publisher MUST ignore any capabilities that it does not recognize.
+	</t>
```

Comment:
May be to prepend "As it is possible the receiver may return custom capability URIs, ..."

PS: unsure if "custom" is the best word.